.changes/v0.8.8.md

```diff
@@ -0,0 +1,15 @@
+## v0.8.8 - 2023-10-11
+
+### Added
+- core: use BuildKit's own readiness implementation instead of our custom logic by @jedevc in https://github.com/dagger/dagger/pull/5808
+- core: add Engine logs for failed queries in debug mode by @jedevc in https://github.com/dagger/dagger/pull/5827
+- core: show GraphQL schema errors in context by @jedevc in https://github.com/dagger/dagger/pull/5874
```

Comment:
The schema merging that causes this error can only be reached by using zenith modules, so I don't think we need to include it here.